datasets/arxiv-2023/metadata.json

```diff
@@ -0,0 +1,41 @@
+{
+    "description": "ARXIV-2023 dataset.",
+    "data": {
+        "Node": {
+            "Titles": {
+                "description": "Title of each node",
+                "type": "str",
+                "format": "Tensor",
+                "file": "arxiv-2023__graph__d37b1b1dcaf915e00432000c09872b62.npz",
+                "key": "Node_Titles"
+            },
+            "Abstracts": {
+                "description": "Abstract of each article(node)",
+                "type": "str",
+                "format": "Tensor",
+                "file": "arxiv-2023__graph__d37b1b1dcaf915e00432000c09872b62.npz",
+                "key": "Node_Abstracts"
+            },
+            "Ids": {
+                "description": "Id of each article(node)",
+                "type": "str",
+                "format": "Tensor",
+                "file": "arxiv-2023__graph__d37b1b1dcaf915e00432000c09872b62.npz",
+                "key": "Node_Ids"
+            }
```

Comment:
The node labels are missing here. You should also convert the `labels` to Attribute and save, too.